src/main/kotlin/scenario.kt

```diff
@@ -0,0 +1,14 @@
+fun scenario() {
+    simulationStep(12,8)
+}
+
+fun simulationStep(numberOfCars : Int, capacity : Int) {
+    val road = Network(capacity)
+    val cars = 1..numberOfCars
+    for (car in cars) {
+        val car = Car(true, road)
+        println("Does this car want to drive? " + car.wantsToDrive)
+        road.overCapacity()
+        println("Is this car ($car) delayed? " + road.analyzeNetwork())
```

Comment:
`analyzeNetwork` is the function that runs the simulation - here you call it once per car. I don't think that's quite what you wanted.